main.py

```diff
@@ -27,8 +25,9 @@
             exit(0)
 
     import requests
-
-    # requests is being importing here because it will error out
+    from src.info import color_text
+    from src.cli.ui import clear as clear_screen
+    # requests clear_screen, and color_text are being importing here because the program will error out
     # if there are missing dependencies in the start of the program
```

Comment:
Maybe this should look something like:

```py
    # requests, clear_screen and color_text are being imported here due to
    # the program throwing an error if there are missing dependencies
    # at the initial start-up phase of the program.
    import requests
    from src.info import color_text
    from src.cli.ui import clear as clear_screen
```

Reply:
Mahas is currently unable to, I shall merge and alter the changes myself.

Thank you, Mahas, for tackling this issue. :+1: